packages/image/types/index.d.ts

```diff
@@ -0,0 +1,24 @@
+import type { SvelteComponentTyped } from 'svelte';
+import type { Img, Picture } from 'vite-imagetools';
```

Comment:
```suggestion
// @ts-ignore since this is an optional peer dependency this might not exist
import type { Img, Picture } from 'vite-imagetools';
```